tests/test_aea.py

```diff
@@ -48,11 +48,15 @@
 from .conftest import CUR_PATH
 
 
-def test_initialise_AEA():
+def test_initialise_aea():
     """Tests the initialisation of the AEA."""
     node = LocalNode()
     address_1 = "address"
-    connections1 = [OEFLocalConnection(address_1, node)]
+    connections1 = [
+        OEFLocalConnection(
+            address_1, node, connection_id=PublicId("fetchai", "local", "0.1.0")
```

Comment:
for programmatic use, the `Connection` subclasses must define the `connection_id`. Feature or design flaw?

Reply:
Design flay maybe? I guess the `connection_id` should be a class variable of the `Connection` abstract class and instantiated in the derived class similar to Protocols?